app/cdap/components/GridTable/components/MenuComponent/mock/mock.ts

```diff
@@ -0,0 +1,3 @@
+export const dummyData = [
```

Comment:
This would need a copyright header but dummy data should just be in the test file that its used since it doesn't look like its shared

Reply:
Done